proto/server.proto

```diff
@@ -25,6 +25,7 @@ message InfoRes {
   string allowed_ips = 7;
   bool dns_enabled = 8;
   string dns_address = 9;
+  int32 mtu = 10;
```

Comment:
The number (`10`) is the address of the field in the communication protocol. It must:
- only exist once,
- and new fileds must be added at the very end of the list

See https://protobuf.dev/programming-guides/dos-donts/#dont-re-use-a-tag-number

> **Don’t Re-use a Tag Number**
Never re-use a tag number. It messes up deserialization. Even if you think no one is using the field, don’t re-use a tag number. If the change was live ever, there could be serialized versions of your proto in a log somewhere. Or there could be old code in another server that will break.

And also

https://protobuf.dev/programming-guides/proto3/#assigning-field-numbers
https://protobuf.dev/programming-guides/proto3/#updating

You thus need to:
- use `15` as the next identifier, and
- move it to the tail of the list, thus:

```
    ...
    string client_config_dns_search_domain = 14;
    int32 mtu = 15;